### ISSUE TITLE
Discontinuation of library

### ISSUE THREAD

**Issue by divmgl** [rakijah/CSGSI#1]
Hello,

Any reason why you chose to discontinue the library? We can definitely improve it together and it would be a nice project to tackle. There are a few things we can do to improve it (I'm almost done with the initial improvements):

* Dependence on `HttpContext`, which can be circumvented using `TcpListener`
* Game state design can be improved to use `Enum` and built-in .NET types
* `UacHelper` needs to be completely removed
* Test project that tests public API 

We can make it much better than @master117's library. Let me know.

**Comment by rakijah** [comment]
Hi,

I've realised that my approach to this was entirely overcomplicated (in it's current state, the library is basically just a wrapper for a JObject).
When I started working on this, I initially used a `TcpListener` instead of the current `HttpListener`. The problem was that ~100 `TcpClient`s were being accepted every second, even though there should have been <1. I switched to the `HttpListener` after I couldn't figure out why this was happening.
Enums were exactly what I was planning to implement to increase usability, but it didn't seem feasible with the current approach.

I'm not really willing to put more energy into this project when I know that there are much more competent developers out there that have already released similar libraries.
